src/content/documentation/develop/manifest-v3-migration-guide.md

```diff
@@ -204,41 +204,59 @@ Also, the `code` parameter is removed so that arbitrary strings can no longer be
 
 To migrate your extension to using non-persistent background pages you need to:
 
-- Update your manifest.json `background` key to remove the `"persistent"` property or set it to `false`.
+- Update your manifest.json `background` key to remove the `"persistent"` property or set it to `false`. This feature is also supported in MV2 since Firefox 106.
 - Ensure listeners are at the top-level and use the synchronous pattern.
 - Record state changes in local storage.
 - Change timers to alarms.
 - Switch from using [`extension.getBackgroundPage`](https://developer.mozilla.org/en-US/Mozilla/Add-ons/WebExtensions/API/extension/getBackgroundPage) to call a function from the background page, to [`runtime.getBackgroundPage`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/runtime/getBackgroundPage).
-- Place menu creation using [`menus.create`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/menus/create) or its alias `contextMenus.create` in a `runtime.onInstalled` listener. Also, note that the `menus.onClicked` event or its alias `contextMenus.onClicked` must be used to handle menu entry click events from an event page. If the `onclick` property of `menus.create` or its alias `contextMenus.create` are used from a call originating from an event page, they throw synchronously. When you migrate an MV2 extension to the event page model, you can use this `onclick` behavior to provide backward-compatible code for Firefox 105 or earlier, where the event pages aren’t supported. For example:
+- Place menu creation using [`menus.create`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/menus/create) or its alias `contextMenus.create` in a `runtime.onInstalled` listener. Also, note that the [`menus.onClicked`](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/menus/onClicked) event or its alias `contextMenus.onClicked` must be used to handle menu entry click events from an event page instead of the `onclick` parameter to the `contextMenus.create` or `contextMenus.update` methods. If the `onclick` property of `menus.create` or its alias `contextMenus.create` are used from a call originating from an event page, they throw synchronously.
 
-  ```javascript
-  eventPagesSupported = false;
-  try {
-    browser.contextMenus.create({ id: "test-menu", onclick: () => {} });
-  } catch (err) {
-    if (err?.message.includes("Property \"onclick\" cannot be used in menus.create")) {
-      // Firefox is detected running with event pages support enabled.
-      eventPagesSupported = true;
-    }
-  } finally {
-    browser.contextMenus.remove("test-menu");
-  }
-  ```
+::: note
+Safari also supports event-driven background scripts, however, Chromium has adopted service workers instead.
+:::
+
+::: note
+Firefox supports non-persistent background pages since Firefox 106. In Firefox 105 and earlier, background pages were always persistent.
```

Comment:
Firefox 105 is in the past, and I wanted to reflect that the state of being unconditionally persistent is past behavior. If "were" is not the right word, would the following alternatives be better?
"were always" -> "had always been" or "used to always be"

Reply:
The point is that you are describing current behavior for 105 (and earlier), "were", "had", "used to" create ambiguity suggesting you are describing some earlier behavior in 105 and it could now be different.

Reply:
I suppose that the difference is that I'm (erroneously?) using Firefox versions as units of dates, while you're looking at them as snapshots of products (identifiable by a incrementing number).

With the suggested edit, it is not immediately obvious that the old behavior is an artifact of the past, and increasingly more obsolete as time passes (and users update to more recent versions).

Do you have a suggestion to improve that?

Reply:
It is because that is what does happen in those versions, not what did happen in those versions (and, therefore, might not happen anymore). How about this
```suggestion
Firefox supports non-persistent background pages from Firefox 106. Firefox supports only persistent background pages in Firefox 105 and earlier.
```